src/extension.ts

```diff
@@ -121,20 +70,61 @@ function decorateEditor(editor: sourcegraph.CodeEditor, sentryProjectId: string,
 }
 
 /**
- * Build URL to the Sentry issues stream page with the Sentry Org, Sentry project ID, and query.
+ * Decorate a line that matches either the line match pattern from the Sentry extension configurations
+ * or that matches common error loggin patterns.
+ * @param i index for decoration range
+ * @param m line match containing error query
+ * @param sentryProjectId Sentry project id retrieved from Sentry extension settings
+ */
+export function decorateLine(
+    i: number,
+    m: RegExpExecArray,
+    sentryProjectId?: string
+): sourcegraph.TextDocumentDecoration {
+    let decoration: sourcegraph.TextDocumentDecoration
```

Comment:
You can declare this as `const` and assign it directly. Always declare things as `const` when possible, and avoid declaring variables as mutable unless absolutely necessary.